pkgs/servers/dendrite/default.nix

```diff
@@ -14,6 +14,27 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-sd2frDxtMd0YUVCCQGaNYU7KopZQC1Ld5OHnjttTrgA=";
 
+  subPackages = [
+    # The server as a monolith: https://github.com/matrix-org/dendrite/blob/main/docs/installation/5_install_monolith.md
+    "cmd/dendrite-monolith-server"
+    # The server as a polylith: https://github.com/matrix-org/dendrite/blob/main/docs/installation/6_install_polylith.md
+    "cmd/dendrite-polylith-multi"
+    # admin tools
+    "cmd/create-account"
+    "cmd/generate-config"
+    "cmd/generate-keys"
+    "cmd/resolve-state"
+    ## seems to be a debugging tool, do we need this?
+    # "cmd/furl"
+    ## an internal tool
+    # "cmd/dendrite-upgrade-tests"
```

Comment:
This is a tool for verifying that dendrite can upgrade from one version to another, and is used in Dendrite's CI infrastructure (https://github.com/matrix-org/dendrite/pull/1901).

Shouldn't be relevant unless you're a Dendrite dev.